packages/flutter/lib/src/material/slider.dart

```diff
@@ -717,6 +726,11 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
     // in range_slider.dart.
     Size screenSize() => MediaQuery.of(context).size;
 
+    Map<ShortcutActivator, Intent> _shortcutMap() =>
```

Comment:
It may be cleaner to use a local variable instead of a local function to get the shortcuts.

Reply:
Done.